plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -124,9 +138,12 @@ public function register_ptk_patterns() {
 			return;
 		}
 
-		$ptk_patterns_loader = new PTKPatternsStore( new PTKClient() );
+		$this->ptk_patterns_store = new PTKPatternsStore( new PTKClient() );
 
-		$patterns = $ptk_patterns_loader->get_patterns();
+		$patterns = $this->ptk_patterns_store->get_patterns();
+		if ( is_wp_error( $patterns ) ) {
+			return;
```

Comment:
If we decided that `get_patterns` will not return any error, we could check if $patterns variable is defined. 

At the same time, it could make sense to add a log here still.

Reply:
Agree! Fixed it here https://github.com/woocommerce/woocommerce/pull/47551/commits/cd1ce72f4ec5cc64b2c1b5aadce877ed40bbdae9